source/_components/switch.neato.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "Neato Robotics Switch"
+description: "Instructions how to integrate a Neato Botvac Connected switch within Home Assistant."
+date: 2016-10-18 16:41
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: neato.png
+ha_category: Switch
+ha_release: 0.3X
+---
+
+The `neato` switch platform allows you to control your [Neato Botvac Connected](https://www.neatorobotics.com/robot-vacuum/botvac-connected-series/botvac-connected/).
+
+To enable this switch in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+    platform: neato
+    username: USERNAME
+    password: PASSWORD
```

Comment:
Under this it's recommended to include a overview for the variables.
Example https://home-assistant.io/components/switch.template/